sklearn/ensemble/_weight_boosting.py

```diff
@@ -314,8 +314,9 @@ class AdaBoostClassifier(ClassifierMixin, BaseWeightBoosting):
 
     learning_rate : float, default=1.
         Learning rate shrinks the contribution of each classifier by
-        ``learning_rate``. There is a trade-off between ``learning_rate`` and
-        ``n_estimators``.
+        ``learning_rate`` whereas a higher learning rate increases the
```

Comment:
actually I don't think that `whereas` is the right conjunction.

Can you just rephrase with

```
Weight applied to each classifier at each boosting iteration. A higher
learning rate increases the contribution of each classifier. There is a
trade-off between the `learning-rate` and `n_estimators` parameters.
```

Reply:
Updated